source/include/dqm4hep/MonitorElementManager.h

```diff
@@ -328,21 +329,21 @@ namespace dqm4hep {
 
       if (!ObjectType::Class()->HasDictionary()) {
         dqm_error("Couldn't book object of type '{0}', because this class has no dictionnary!",
-                  ObjectType::Class_name());
+                  ObjectType::Class_Name());
```

Comment:
Looking at the TObject documentation, TObject doesn't have a `Classe_Name()` function, but `ClassName()`
Did you reimplement it somehere?

Reply:
This method comes from the ClassDef macro of each ROOT class. By doing so, we also ensure that every booked object is implemented with a ClassDef macro, thus have a streamer method implemented (coming with ClassDef too) and thus can be written in json using the ROOT streaming facility.

Reply:
Okay, never really looked into the `ClassDef` macro before. Good point again.

Reply:
Just to keep track of it : 
https://root.cern.ch/doc/v608/Rtypes_8h.html#af17c75f6ae6ab99b7977b679be5f147b